lib/backgrounder/workers/process_asset_mixin.rb

```diff
@@ -11,8 +11,9 @@ def self.included(base)
 
       def perform(*args)
         record = super(*args)
+        return if record.nil?
```

Comment:
Even though this solves the issue in questions, I'd say that the job should not swallow these exceptions, if there was an exception while loading a record, I think the job should fail or somehow let people know something went wrong.